src/constants.py

```diff
@@ -0,0 +1,14 @@
+n_jobs = 4
```

Comment:
naming for constants - it should be upper case everywhere 
and let's delete this constant - let's specify default value for `argparses`/`click`